ansible.cfg.template

```diff
@@ -9,10 +9,22 @@
 # the run, so you can still validate what would be output.
 show_custom_stats = true
 
-# This will be necessary until we get the fips_assessment collection going
-roles_path = ./roles:../fips-assessments/roles
```

Comment:
if we are removing the `roles_path` here, don't we need to also modify the use of `ansible.builtin.include_role`? 

https://github.com/opdev/opcap-ansible/blob/bb5badcead74c30839633e86ad69fa4480f2d93c/roles/fips_assessment/tasks/main.yml#L35C1-L35C1

wasn't sure if you were intending to call the fips-assessment collection roles via a collection module or just copy the roles over.

Reply:
The collection gets installed into the ee. I'll make sure this is still working as expected without the roles path.